tests/docker-build

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+set -e
+
+raspbian_version="2019-04-08"
+
+scriptdir=$(cd `dirname $0` && pwd)
+
+(cd ${scriptdir}/docker-files && ./prepare-raspbian-img.sh ${raspbian_version})
+
+docker build --no-cache --build-arg raspbian_version=${raspbian_version} -t mender-dist-packages-tester ${scriptdir}/docker-files/
```

Comment:
Btw, why `--no-cache`?

Reply:
It sholdn't be! I was experimenting locally and commit that by mistake in one of the rebases. Good catch, I'll fix